test-app/runtime/src/main/cpp/CallbackHandlers.cpp

```diff
@@ -634,7 +634,7 @@ void CallbackHandlers::LogMethodCallback(const v8::FunctionCallbackInfo<v8::Valu
 }
 
 void CallbackHandlers::TimeCallback(const v8::FunctionCallbackInfo<v8::Value>& args) {
-    auto nano = std::chrono::time_point_cast<std::chrono::nanoseconds>(std::chrono::steady_clock::now());
+    auto nano = std::chrono::time_point_cast<std::chrono::nanoseconds>(std::chrono::system_clock::now());
     double duration = nano.time_since_epoch().count() / 1000000.0;
```

Comment:
wouldn't it be easier to just do:
```
auto timeNow = std::chrono::system_clock::now().time_since_epoch()
auto duration = std::chrono::time_point_cast<std::chrono::milliseconds>(timeNow);
````

Reply:
A valid point, we should not be avoiding the type system imposed by `chrono`.